pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1078,9 +1097,27 @@ BEGIN
 
         --chunks where the end time is before now()-1 hour will be compressed
         IF SCHEMA_CATALOG.get_timescale_major_version() >= 2 THEN
-            PERFORM add_compression_policy(format('SCHEMA_DATA.%I', metric_table_name), INTERVAL '1 hour');
+            IF SCHEMA_CATALOG.is_multinode() THEN
```

Comment:
should we only do this for multinode? My feeling is do this for all of 2.0

Reply:
let's merge this then throw the switch

Reply:
why are we waitting?